plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/assign_user_to_investigation/schema.py

```diff
@@ -68,28 +68,18 @@ class AssignUserToInvestigationOutput(insightconnect_plugin_runtime.Output):
     }
   },
   "required": [
+    "created_time",
+    "disposition",
     "investigation",
-    "success"
+    "last_accessed",
+    "organization_id",
+    "priority",
+    "rrn",
+    "source",
+    "status",
+    "title"
```

Comment:
As investigation is the only property here, having the extra that are added from the refresh plugin will cause issues 
```
"required": [
    "investigation"
]
```